app/src/main/kotlin/com/wire/android/core/functional/Either.kt

```diff
@@ -0,0 +1,103 @@
+package com.wire.android.core.functional
```

Comment:
In this case it is me, but I would like to keep the original @author of every piece of code we are re-using from other projects. Just to give credit to who deserves. Open source and contributions for the win!

Reply:
oops. mistook it for a regular copyright. added!